src/2_resources.js

```diff
@@ -93,7 +93,8 @@ resources.open = {
 	endpoint: "/v1/open",
 	method: utils.httpMethod.POST,
 	params: {
-		"browser_fingerprint_id": validator(true, branch_id),
+		"browser_fingerprint_id": validator(false, branch_id),
+		"alternative_browser_fingerprint_id": validator(false, branch_id),
```

Comment:
@zeeshabnc this file just defines the parameter type and validates the request. If you look at where open is called in `6_branch` you will see a difference and that `alternative_browser_fingerprint_id` is the value from storage